tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,321 @@
+
+_Possible answers: Met/Unmet_
+
+> The project MUST provide a "quick start" guide for new users to help them quickly do something with the software. (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+
+The project MUST make an effort to keep the documentation consistent with the current version of the project results (including software produced by the project). Any known documentation defects making it inconsistent MUST be fixed. If the documentation is generally current, but erroneously includes some older information that is no longer true, just treat that as a defect, then track and fix as usual.
+
+_Possible answers: Met/Unmet/NA_
+
+
+The project repository front page and/or website MUST identify and hyperlink to any achievements, including this best practices badge, within 48 hours of public recognition that the achievement has been attained. (URL required) 
+
+_Possible answers: Met/Unmet_
+
+
+## Accessibility and internationalization
+
+> The project (both project sites and project results) SHOULD follow accessibility best practices so that persons with disabilities can still participate in the project and use the project results where it is reasonable to do so.
+
+_Possible answers: Met/Unmet/NA_
+
+> The software produced by the project SHOULD be internationalized to enable easy localization for the target audience's culture, region, or language. If internationalization (i18n) does not apply (e.g., the software doesn't generate text intended for end-users and doesn't sort human-readable text), select "not applicable" (N/A)
+
+_Possible answers: Met/Unmet/NA_
+
+
+## Other
+
+> If the project sites (website, repository, and download URLs) store passwords for authentication of external users, the passwords MUST be stored as iterated hashes with a per-user salt by using a key stretching (iterated) algorithm (e.g., Argon2id, Bcrypt, Scrypt, or PBKDF2). If the project sites do not store passwords for this purpose, select "not applicable" (N/A).
+
+_Possible answers: Met/Unmet/NA_
+
+
+
+# Change Control
+
+## Previous versions
+
+> The project MUST maintain the most often used older versions of the product or provide an upgrade path to newer versions. If the upgrade path is difficult, the project MUST document how to perform the upgrade (e.g., the interfaces that have changed and detailed suggested steps to help upgrade).
+
+_Possible answers: Met/Unmet/NA_
+
+
+# Reporting
+
+## Bug-reporting process
+
+> The project MUST use an issue tracker for tracking individual issues. 
+
+**Met**
+https://github.com/nodejs/node/issues
+
+## Vulnerability report process
+
+> The project MUST give credit to the reporter(s) of all vulnerability reports resolved in the last 12 months, except for the reporter(s) who request anonymity. If there have been no vulnerabilities resolved in the last 12 months, select "not applicable" (N/A). (URL required)
+
+_Possible answers: Met/Unmet/NA_
```

Comment:
This is integrated in Hacker One, [Node.js hacktivity](https://hackerone.com/nodejs/hacktivity)
```suggestion
> The project MUST give credit to the reporter(s) of all vulnerability reports resolved in the last 12 months, except for the reporter(s) who request anonymity. If there have been no vulnerabilities resolved in the last 12 months, select "not applicable" (N/A). (URL required)

**Met**
```

Reply:
Met